front/src/plugins/i18n.js

```diff
@@ -0,0 +1,11 @@
+﻿import Vue from 'vue';
+import VueI18n from 'vue-i18n';
+import messages from '@/locales';
```

Comment:
you can add `@locals` path in vue.config

Reply:
have added `@locales` to vue.config